src/include/OSL/llvm_util.h

```diff
@@ -88,6 +89,9 @@ class OSLEXECPUBLIC LLVM_Util {
               int vector_width = 4);
     ~LLVM_Util();
 
+    // map of function names and prefixes that we would like prepended to these functions
+    std::unordered_map<ustring, ustring, ustringHash> m_function_prefixes;
```

Comment:
Recent changes in the code base have made this declaration change, the ustringHash is no longer necessary.

```suggestion
    std::unordered_map<ustring, ustring> m_function_prefixes;
```